### PR TITLE
Fix the daily run of freshclam

### PR DESCRIPTION
**Pull request by dazoakley**

### CODE REVIEW

File: hmpps-clamav/Dockerfile.freshclammed [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-USER "root"
+USER root
+RUN chown -R clamav:clamav /var/lib/clamav
+
+USER clamav
 RUN freshclam
 
 # We run as UID 1000 in kubernetes, this will allow us to start the process
+USER root
 RUN mkdir -p /var/run/clamav && \
```
Comment (steverendell) [inline hmpps-clamav/Dockerfile.freshclammed:11]: You actually dont need anything in this dockerfile except RUN freshclam.
Running freshclam as root overwrote the correct ownership of the dir

**Comment by swilson-mojhub** [comment]
Isn't there a restriction on running processes at `root` in CP?

File: hmpps-clamav/Dockerfile.freshclammed (hunk 1 above)
Comment (dazoakley) [reply hmpps-clamav/Dockerfile.freshclammed:11]: Oh cool, if that's the case, let's go with that! 😄

**Comment by dazoakley** [comment]
@steverendell has a much better fix

**Pull request closed without merging**